.gitignore

```diff
@@ -1,6 +1,6 @@
 *.pyc
 *.egg-info
-*.toml
+/*.toml
```

Comment:
`*.toml` is ignored because we don't want to commit secret tokens accidentally. But inside `reconcile/test/fixtures` there are some toml files that need to be commited, that's the reason for this change.